packages/ui/src/composables/useColors.ts

```diff
@@ -42,18 +42,20 @@ export const useColors = () => {
     throw new Error('useColors must be used in setup function or Vuestic GlobalConfigPlugin is not registered!')
   }
 
-  const { setGlobalConfig, globalConfig } = gc
+  const { globalConfig } = gc
 
-  const colors = useReactiveComputed<ColorVariables>({
+  let colors = useReactiveComputed<ColorVariables>({
     get: () => globalConfig.value.colors!.variables,
-    set: (v: ColorVariables) => { setColors(v) },
+    set: (colorVars: Partial<ColorVariables>) => {
+      globalConfig.value.colors!.variables = {
+        ...globalConfig.value.colors.variables,
+        ...colorVars,
+      } as ColorVariables
+    },
   })
 
-  const setColors = (colors: Partial<ColorVariables>): void => {
-    globalConfig.value.colors!.variables = {
-      ...globalConfig.value.colors.variables,
-      ...colors,
-    } as ColorVariables
+  const setColors = (colorVars: Partial<ColorVariables>): void => {
+    colors = colorVars
```

Comment:
This bailed a warning: 
`Partail<ColorVarialbes>` cannnot assign to `ColorVariables`.
Does `colors = colorvars` not calling `set`?
What should I do to fix this warning?